ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/WorkloadCVEOverviewTable.tsx

```diff
@@ -384,6 +397,12 @@ function WorkloadCVEOverviewTable({
                                         >
                                             <DateDistance date={firstDiscoveredInSystem} />
                                         </Td>
+                                        <Td
+                                            dataLabel="Published"
+                                            className={getVisibilityClass('publishedOn')}
```

Comment:
How about `modifier="nowrap"` prop as in DeploymentVulnerabilitiesTable.tsx file?

Picture at 1315px width:
![WorkloadCVEOverviewTable_nowrap_1315px](https://github.com/user-attachments/assets/677b47cd-738b-46dd-8fbe-61f6b741e6ef)